### ISSUE TITLE
[Bug] GM_openInTab doesn't open as child in Tree Style Tab

### ISSUE THREAD

**Issue by hanjo** [Tampermonkey/tampermonkey#1678]
Hi,

I've created a userscript that opens a tab in the background using the following code:
```js
GM_openInTab('https://google.com', { loadInBackground: true, setParent: true })
```
Unfortunately, the tab is not opened as a child to the current tab with Tampermonkey (v4.18.1) on Firefox (v108.0.1), instead the tab is opened to the right of the current tab on the same level - ignoring the container of the parent tab, but that's probably another issue.

I have opened a bug over at Tree Style Tab. There, it's been explained that the issue is with Tampermonkey, since Tampermonkey has to open a new tab with its opener information using the `openerTabId`:
https://github.com/piroor/treestyletab/issues/3272

Can this be implemented?

Thanks!

**Comment by derjanb** [comment]
> Can this be implemented?

It should be implemented. I'll double check...

**Comment by hanjo** [comment]
Hi Jan,

thank you for your efforts. I've installed 4.19.6177, imported my userscript and disabled the stable Tampermonkey. I then restarted Firefox (just to be sure), but the behavior seems to be the same as before. Tabs are still opening to the right (not as child) without any container assigned.

Anything I would need to clear at my end?

**Comment by derjanb** [comment]
I see. Its my fault.

**Comment by derjanb** [comment]
This should be fixed at 4.19.6178 ([xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

Please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons`.

**Comment by hanjo** [comment]
I can confirm this works now in 4.19.6178. Thank you!
